testutils/gke/client.go

```diff
@@ -95,8 +96,8 @@ func (gsc *sdkClient) DeleteClusterAsync(project, region, zone, clusterName stri
 	if zone != "" {
 		return gsc.Projects.Zones.Clusters.Delete(project, location, clusterName).Context(context.Background()).Do()
 	}
-	parent := fmt.Sprintf("projects/%s/locations/%s/clusters/%s", project, location, clusterName)
-	return gsc.Projects.Locations.Clusters.Delete(parent).Context(context.Background()).Do()
+	opsFullPath := fmt.Sprintf("projects/%s/locations/%s/clusters/%s", project, location, clusterName)
```

Comment:
this one should be `clusterFullPath` still, based on the last word in it's path

Reply:
Done.